akka-stream/src/main/scala/akka/stream/Attributes.scala

```diff
@@ -297,12 +297,18 @@ object ActorAttributes {
 
   /**
    * Scala API: Decides how exceptions from user are to be handled.
+   *
+   * Note that this is implemented on a per stage basis. Stages supporting supervision has
```

Comment:
"have"

though perhaps more explicitly

`Stages supporting supervision strategies explicitly document that they do so. If a stage does not document support for these, it should be assume to not support supervision.`